test/functional/page_objects/common_page.ts

```diff
@@ -549,7 +549,7 @@ export class CommonPageObject extends FtrService {
    * @param time
    */
   async setTime(time: TimeStrings) {
-    await this.kibanaServer.uiSettings.replace({
+    await this.kibanaServer.uiSettings.update({
```

Comment:
So replace was causing default data view to get unset?

Reply:
yep. Works fine now and also solves few other test failures.